src/agent/onefuzz/src/asan.rs

```diff
@@ -55,6 +59,16 @@ impl AsanLog {
     }
 }
 
+fn parse_failure(text: &str) -> Option<(String, String, String)> {
```

Comment:
Can we rename this to `parse_asan_runtime_error()` or similar, to make the difference between this an "the target code failed" extra obvious?